projects/organization-management/src/app/shared/user/user-profile-form/user-profile-form.component.ts

```diff
@@ -0,0 +1,13 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'ish-user-profile-form',
+  templateUrl: './user-profile-form.component.html',
+  changeDetection: ChangeDetectionStrategy.Default,
+})
+export class UserProfileFormComponent {
+  @Input() form: FormGroup;
+  @Input() titles: string[];
```

Comment:
determine titles im this component and remove logic from the create and edit pages, this input parameter should not be necessary, because we are allowed to read data via the facades also in dumb components now